src/main/java/com/edmunds/rest/databricks/DTO/scim/group/GroupDTO.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2018 Edmunds.com, Inc.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+package com.edmunds.rest.databricks.DTO.scim.group;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.Arrays;
+import java.util.Objects;
+import lombok.Data;
+
+/**
+ * Databricks SCIM Group.
+ *
+ * @see <a href="https://tools.ietf.org/html/rfc7643#section-4.2">https://tools.ietf.org/html/rfc7643#section-4.2</a> .
+ */
+@Data
+public class GroupDTO {
+
+  @JsonProperty("schemas")
+  private final String[] schemas = new String[]{"urn:ietf:params:scim:schemas:core:2.0:Group"};
+  @JsonProperty("id")
+  private long id;
+  @JsonProperty("displayName")
+  private String display;
+  @JsonProperty("members")
+  private MemberDTO[] memberDTOS = new MemberDTO[0];
+
+  public GroupDTO() {
+  }
+
+  /**
+   * Creates a group from another one.
+   * @param from object to copy from
+   */
+  public GroupDTO(GroupDTO from) {
+    this.id = from.id;
+    this.display = from.display;
+    if (from.memberDTOS != null) {
+      memberDTOS = Arrays.copyOf(from.memberDTOS, from.memberDTOS.length);
+    }
+  }
+
+  @Override
```

Comment:
As far as we already have Lombok dependency, can we use here `@EqualsAndHashCode` annotation instead of these two methods? (it allows to choose which fields shall be used for computations)

P.S. `@Data` annotation already provides this one but looks like default behavior doesn't suit us
https://projectlombok.org/features/Data

Reply:
@Data can't be used because of the weird (my opinion) way the user creation works right now. For example you can't overwrite the email (it's always the username) and you can only overwrite only some attributes specified in the SCIM schema=>it would be misleading to allow setters for all the properties.
I've created this ticket https://github.com/MicrosoftDocs/azure-docs/issues/43491 btw.
On the other hand using the lombok annotations for eq/hash/tostring seems a good idea and is more aligned with the existing style so I'll change the code.

Reply:
But there is `@Data` annotation in that class already, probably you've meant UserDTO class?
Then you have 2 options to reduce boilerplate code volume:
1) use `@Getter @Setter` annotation pair for particular fields only
2) use `@Data` and exclude getters/setters generation for fields requires custom approach:
https://stackoverflow.com/a/54782694

Reply:
Thanks, I've used the second approach.